### DISCUSSION TITLE
Config setup help - SOLVED

### DISCUSSION THREAD

**Discussion by armeya25** [BxNxM/micrOS#21] [General]
How do I set default IP adress to 192.168.0.125
Fresh installation set it to 192.188.4.1
My router ip is  192.168.0.1

**Comment by BxNxM** [comment]
Hi, you can just overwrite it in the config, it will be set as default IP after reboot -h (hard reset)
So my recommendation:
1.       Connect to the board over IP (telnet or devToolKit.py -c)
2.       conf
3.       devip 192.168.0.125
4.       noconf
5.       reboot -h
then after device reboot it will use  192.168.0.125 if it was not occupied when device boots up.

What version of micrOS are you using? (I am jsut curious, the AP mode should not rewrite the devip parameter in the configuration).

**Comment by BxNxM** [comment]
BTW, sorry for the late reply, I haven't seen any notification about your question in github app, but I will check GH discussion more frequently :)
